ci/jenkins/scripts/test-aws.sh

```diff
@@ -140,15 +146,22 @@ ci/kind/kind-setup.sh create kind
 
 # Create a key pair
 KEY_PAIR="nephe-$$"
-aws ec2 import-key-pair --key-name ${KEY_PAIR} --public-key-material fileb://~/.ssh/id_rsa.pub --region ${AWS_DEFAULT_REGION}
+aws ec2 import-key-pair --key-name ${KEY_PAIR} --public-key-material fileb://~/.ssh/id_rsa.pub --region "${AWS_DEFAULT_REGION}"
 
 export TF_VAR_aws_key_pair_name=${KEY_PAIR}
 
 function cleanup() {
     # Delete key pair
-    aws ec2 delete-key-pair  --key-name ${KEY_PAIR}  --region ${AWS_DEFAULT_REGION}
+    aws ec2 delete-key-pair  --key-name ${KEY_PAIR}  --region "${AWS_DEFAULT_REGION}"
 }
 trap cleanup EXIT
 
-mkdir -p $HOME/logs
-ci/bin/integration.test -ginkgo.v -ginkgo.timeout 90m -ginkgo.focus=".*test-aws.*" -kubeconfig=$HOME/.kube/config -cloud-provider=AWS -support-bundle-dir=$HOME/logs
+mkdir -p "$HOME"/logs
+
+if [ "$UPGRADE" = true ] ; then
```

Comment:
Would it make more sense to make upgrades a separate focus, like `test-upgrade-aws/azure` instead of a separate suite?

Reply:
We cannot be that.. Upgrade suite doesnt take nephe manifest. It start with 0.5 version and then upgrade to TOT. mixing with integration wasn't clean.